cmd/nuntium/mediator.go

```diff
@@ -186,7 +186,11 @@ func (mediator *Mediator) getMRetrieveConf(mNotificationInd *mms.MNotificationIn
 		log.Print("Download issues: ", err)
 		return
 	} else {
-		storage.UpdateDownloaded(mNotificationInd.UUID, filePath)
+		err := storage.UpdateDownloaded(mNotificationInd.UUID, filePath)
```

Comment:
nice catch, can I recomment:

if err := ....; err != nil {
    log....
    return
}

to contain that err var?